mslice/plotting/plot_window/plot_figure_manager.py

```diff
@@ -87,14 +85,18 @@ def flag_as_current(self):
 
     def add_slice_plot(self, slice_plotter_presenter, workspace):
         if self._plot_handler is None:
-            self.move_window(-self.window.width() / 2, 0)
+            # Move the top right corner of all slice plot windows to the left of the screen centre by 1.05
```

Comment:
This is a good description of what is being done but could we add a description of why - that's actually more important for the future.